pkg/synchronizer/synchronizer.go

```diff
@@ -338,42 +345,48 @@ func (n *Synchronizer) Prepare(app *nais_io_v1alpha1.Application) (*Rollout, err
 }
 
 // ClusterOperations generates a set of functions that will perform the rollout in the cluster.
-func (n *Synchronizer) ClusterOperations(ctx context.Context, rollout Rollout) []func() error {
-	var fn func() error
-
-	funcs := make([]func() error, 0)
-	deletes := make([]func() error, 0)
+func (n *Synchronizer) ClusterOperations(ctx context.Context, rollout Rollout) []commit {
+	funcs := make([]commit, 0)
+	deletes := make([]commit, 0)
 
 	for _, rop := range rollout.ResourceOperations {
+		c := commit{
+			groupVersionKind: rop.Resource.GetObjectKind().GroupVersionKind(),
```

Comment:
Har dere verifisert at dette gir en verdi som er god og beskrivende? Jeg mener jeg har vært borti tilfeller hvor den blir blank/har noe rare artifakter.

Reply:
Nei, tok det for gitt, skal gjøre en bedre sjekk på det. Takk for innspill :)

Reply:
Gjorde en test med en minimal app i lokalt kluster, alle ressursene returnerte en kind som forventet. Men har lagt til en liten ekstra sjekk for å unngå eventuelle `nil`-problemer.